### ISSUE TITLE
False negative for unpacked arguments

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#10060]
Pyright doesn't correctly flag errors when an unpacked operator is used in an argument expression for a call and the unpacked type is not iterable.

```python
def func(x: None):
    print(*x) # Should be an error
```

This is related to mypy issue https://github.com/python/mypy/issues/18783.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.397.
